test/NUnit.Portable.Agent.Tests/NUnitPortableDriverTests.cs

```diff
@@ -49,14 +50,6 @@ public void CreateDriver()
             _driver = new NUnitPortableDriver();
         }
 
-        #region Construction Test
-
-        public void ConstructController_MissingFile_ThrowsArgumentInvalid()
-        {
-            Assert.That(new NUnitPortableDriver(), Throws.ArgumentException);
-        }
-        #endregion
-
```

Comment:
I presume this is old code? The method wasn't run as a test and doesn't seem relevant to the current implementation - I removed it.